Badge/badge.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<script src="https://kit.fontawesome.com/bbbaa28fd5.js" crossorigin="anonymous"></script>
+		<link rel="stylesheet" href="../Avatar/avatar.css" />
+		<link rel="stylesheet" href="badge.css" />
+		<title>Badge</title>
+	</head>
+	<body>
+		<h1>Badge on Avatar</h1>
+		<div class="badge-container">
+			<img src="../Assets/avatar-2.jpg" alt="user avatar" class="avatar avatar-lg" />
```

Comment:
I am not able to view Badge on Avatar images in CSB. Please check. Maybe you have added your assets folder in gitignore so it is giving an error in console probably.

Reply:
Same with Avatar component.

Reply:
Thank you for the review Raksha, I removed the file from `.gitignore`, can you check it now?